src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int firstLeg;
+    private int secondLeg;
+
+    public RightTriangle(int firstLeg, int secondLeg, String color) {
+        super(color);
+        this.firstLeg = firstLeg;
+        this.secondLeg = secondLeg;
+    }
+
+    @Override
+    public double getArea() {
+        return (double) firstLeg * secondLeg / 2;
+    }
+
+    @Override
+    public void draw() {
```

Comment:
Avoid using `System.out.println` for drawing figures. It is better to return the drawing result as a String and let the caller decide how to display or handle it.